cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/AnalyzerState.java

```diff
@@ -37,16 +39,28 @@ public boolean proposalReady() {
    * Return an object that can be further used
    * to encode into JSON
    */
-  public Map<String, Object> getJsonStructure() {
+  public Map<String, Object> getJsonStructure(boolean verbose) {
     Map<String, Object> analyzerState = new HashMap<>();
     Set<String> readyGoalNames = new HashSet<>();
+    List<Object> goalReadinessList = new ArrayList<>();
```

Comment:
Maybe we can optimize this by setting the `initialCapcity` for the `goalReadinessList` to `_readyGoals.size()`